File: Topic Based Problems/01 Fundamentals/1 Beginner/06 Rectangle Area/Java/RectangleArea.java
import java.util.Scanner;

public class RectangleArea {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int length, width;

        System.out.print("Enter length: ");
        length = scanner.nextInt();

        System.out.print("Enter width: ");
        width = scanner.nextInt();

        System.out.println("Area of rectangle is: " + (length * width));

        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/06 Rectangle Area/C/RectangleArea.c
#include <stdio.h>

int main() {
    int length, width;

    printf("Enter length: ");
    scanf("%d", &length);

    printf("Enter width: ");
    scanf("%d", &width);

    printf("Area of rectangle is: %d\n", length * width);

    return 0;
}
